### ISSUE TITLE
Flag multiple installation steps

### ISSUE THREAD

**Issue by jisantuc** [hadolint/hadolint#373]
- ~This is a bug report~
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

```dockerfile
RUN pip install rasterio==1.0.18
RUN pip install matplotlib==3.2.0
```

should produce:

```
Consolidate installation steps to reduce layers
```

### Actual behavior

does nothing

**Comment by jisantuc** [comment]
I'm happy to give this a shot if you agree it's a good idea. I only learned this tool exists earlier today and I already :heart: it

**Comment by zemanlx** [comment]
Sounds good to me. I can not imagine a reason why this should not be caught as an issue in any production image. It would be great if you can make a PR by yourself.
